### PR TITLE
aten::where aten::nonzero aten::nonzero_numpy

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: src/frontends/pytorch/src/op/where.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    auto cond = context.get_input(0);
+    if (context.input_is_none(1)) {
+        auto non_zero_cond = context.mark_node(std::make_shared<opset8::NonZero>(cond, element::i64));
+        auto unsqueezed_rank = context.mark_node(get_rank_node(cond));
+        auto rank = context.mark_node(std::make_shared<opset8::Squeeze>(unsqueezed_rank));
+        auto one = context.mark_node(opset8::Constant::create(element::i64, Shape{}, {1}));
+        auto zero = context.mark_node(opset8::Constant::create(element::i64, Shape{}, {0}));
+        auto split_lens = context.mark_node(std::make_shared<opset8::Broadcast>(one, unsqueezed_rank));
+        auto tuple_results = std::make_shared<opset8::VariadicSplit>(non_zero_cond, zero, split_lens)->outputs();
+        for (size_t i = 0; i < tuple_results.size(); i++){
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/where.cpp]: Please fix codestyle

**Review by mvafin — CHANGES_REQUESTED** [review]

File: src/frontends/pytorch/src/op/where.cpp (hunk 1 above)
Comment (eaidova) [reply src/frontends/pytorch/src/op/where.cpp]: fixed

File: src/frontends/pytorch/src/transforms/prim_list_unpack_replacer.cpp [hunk 2]
```diff
@@ -101,7 +119,24 @@ PrimListUnpackReplacer::PrimListUnpackReplacer() {
 
             return true;
         }
+        if (auto where = cast_fw_node(input_node, "aten::nonzero_numpy")) {
```
Comment (mvafin) [inline src/frontends/pytorch/src/transforms/prim_list_unpack_replacer.cpp]: Better to rename `where` variable here, it looks like the name is just a copy artifact.

Comment (eaidova) [reply src/frontends/pytorch/src/transforms/prim_list_unpack_replacer.cpp]: renamed

**Comment by eaidova** [comment]
@mvafin is there anything else what should be done?

**Review by mvafin — APPROVED** [review]

**Pull request merged**